mempool/cat/reactor_test.go

```diff
@@ -0,0 +1,341 @@
+package cat
+
+import (
+	"encoding/hex"
+	"os"
+	"sort"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/go-kit/log/term"
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/tendermint/abci/example/kvstore"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	p2pmock "github.com/tendermint/tendermint/p2p/mock"
+
+	cfg "github.com/tendermint/tendermint/config"
+
+	"github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/mempool"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/p2p/mocks"
+	protomem "github.com/tendermint/tendermint/proto/tendermint/mempool"
+	"github.com/tendermint/tendermint/proxy"
+	"github.com/tendermint/tendermint/types"
+)
+
+const (
+	numTxs  = 10
+	timeout = 120 * time.Second // ridiculously high because CircleCI is slow
```

Comment:
outdated comment? We aren't using Circle CI right?

Reply:
Yeah this is copied from the existing tendermint v1 reactor_test